lesson_06/task_3/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"text/template"
+	"time"
+)
+
+type response struct {
+	Cookie []string
+}
+
+func handlerCookie(w http.ResponseWriter, r *http.Request) {
+	// Web server, responds to “/”
+	if r.URL.Path != "/" {
+		http.Error(w, "404 not found.", http.StatusNotFound)
+		return
+	}
+
+	switch r.Method {
+	case "GET":
+		// get value from Header[Cookie]
+		var tm response
+		x1 := r.Header["Cookie"]
+		tm.Cookie = x1
+		// load html witch data
+		tmpl := template.Must(template.ParseFiles("form.html"))
+		tmpl.Execute(w, tm)
+
+	case "POST":
```

Comment:
the same is here, use constants please
https://pkg.go.dev/net/http#pkg-constants